specs/ipyplotly_integration/Overview.ipynb

```diff
@@ -0,0 +1,909 @@
+   "execution_count": 31,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "scatt1.on_selected(brush)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Now box or lasso select points on the figure and see them turn red"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 32,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Reset brush\n",
+    "selected = np.zeros(iris_class.size)\n",
+    "scatt1.marker.color = selected"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Create second plot with different features"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 38,
+   "metadata": {
+    "scrolled": false
+   },
+   "outputs": [
+    {
+     "data": {
+      "application/vnd.jupyter.widget-view+json": {
+       "model_id": "e88d348677544c1ea2eca95ea8b30fdd",
+       "version_major": 2,
+       "version_minor": 0
+      },
+      "text/html": [
+       "<p>Failed to display Jupyter Widget of type <code>FigureWidget</code>.</p>\n",
+       "<p>\n",
+       "  If you're reading this message in the Jupyter Notebook or JupyterLab Notebook, it may mean\n",
+       "  that the widgets JavaScript is still loading. If this message persists, it\n",
+       "  likely means that the widgets JavaScript library is either not installed or\n",
+       "  not enabled. See the <a href=\"https://ipywidgets.readthedocs.io/en/stable/user_install.html\">Jupyter\n",
+       "  Widgets Documentation</a> for setup instructions.\n",
+       "</p>\n",
+       "<p>\n",
+       "  If you're reading this message in another frontend (for example, a static\n",
+       "  rendering on GitHub or <a href=\"https://nbviewer.jupyter.org/\">NBViewer</a>),\n",
+       "  it may mean that your frontend doesn't currently support widgets.\n",
+       "</p>\n"
+      ],
+      "text/plain": [
+       "FigureWidget()"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "f2 = FigureWidget(data=[{'type': 'scatter',\n",
+    "                         'x': iris_df.petal_length, \n",
+    "                         'y': iris_df.sepal_width,\n",
+    "                         'mode': 'markers'}])\n",
+    "f2"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 39,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Set axis titles\n",
+    "f2.layout.xaxis.title = 'petal_length'\n",
+    "f2.layout.yaxis.title = 'sepal_width'"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 40,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Grab trace reference\n",
+    "scatt2 = f2.data[0]"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 41,
+   "metadata": {
+    "scrolled": false
+   },
+   "outputs": [],
+   "source": [
+    "# Set marker styles / colorbars to match between figures\n",
+    "scatt2.marker = scatt1.marker"
```

Comment:
not a big deal, but running this cell more than once raises a `ValueError `:
```
ValueError: The truth value of an array with more than one element is ambiguous. Use a.any() or a.all()
```

I suggest changing the code so that an error doesn't get raised or writing a little blurb about the reason why this happens.

Reply:
Fixed